tests/testsuite.src/syn_misc.at

```diff
@@ -8385,3 +8385,52 @@ prog.cob:18: error: ANY LENGTH items may only be BY REFERENCE formal parameters
 
 AT_CLEANUP
 
+AT_SETUP([IMP INCLUDE directive])
```

Comment:
please use a double empty line before `AT_SETUP` to structure the file a bit better (also for the other file)